### ISSUE TITLE
[WF] Images show as plain text

### ISSUE THREAD

**Issue by bowshewicz** [Infragistics/convertwire#155]
Topic: win-windows-forms-controls-and-components.adoc
Staging Link: http://help.staging.infragistics.local/doc/winforms?page=Win_Windows_Forms_Controls_and_Components.html

The images in the table appear as plain text

Here's what I see:
![image](https://cloud.githubusercontent.com/assets/16781726/14544955/a68e9690-0269-11e6-84ac-17904393b928.png)

And here's what it looks like on the staging site:
![image](https://cloud.githubusercontent.com/assets/16781726/14545006/e6653cba-0269-11e6-89ae-4da0306541e0.png)

**Comment by ig-ealbert** [comment]
It seems to work correctly using all inline images (image:Images/...).
I did this as a fix to the ReportPlus Desktop (Japanese) help before discussing the issue with Craig.  It seems to work, as I changed all the images to inline syntax.
http://help.jp.staging.infragistics.local/reportplus/getting-started

**Comment by craigshoemaker** [comment]
I noticed two issues with the topic in question. 

First, in one of the cells there is an image `src` value with a space in it. AsciiDoc does not like this, so I have added a check in the reporting to look for any topic that has images with spaces in the paths so we can fix it.

Second, for some reason one of cells (the same cell) has a link on a new line with an extra space which throws off the rendering of the cell contents. I was able to adjust the cell rendering logic to compensate for this issue.

The all the other cells in this topic appear to be rendering correctly in HTML so I am going to call this issue closed by a1dda14a0ee2121793c83771747aeb0c72f2bc1d
